package.json

```diff
@@ -85,7 +86,15 @@
                 "azure-pipelines.configure": {
                     "type": "boolean",
                     "default": true,
-                    "description": "Enable 'Configure Pipeline' feature"
+                    "description": "Enable 'Configure Pipeline' feature",
+                    "order": 0
+                },
+                "azure-pipelines.1ESPipelineTemplatesSchemaFile": {
+                    "type": "boolean",
+                    "default": false,
```

Comment:
Why not true by default? :D

Reply:
We want customers to consciously set it to true